galette/lib/Galette/Core/Galette.php

```diff
@@ -178,23 +178,22 @@ public static function getMenus(bool $public = false): array
                         'title' => _T("View, search into and filter member's list"),
                         'route' => [
                             'name' => 'members',
-                            'args' => []
+                            'aliases' => ['editMember'],
+                            'sub_select' => false
```

Comment:
I cannot choose beetween members list or member add, no one seems relevant. So, no sub menu will be selected.

Reply:
Members can be edited from the list. In my opinion, it's relevant enough. Or at least, more relevant than member add, don't you think ?

Reply:
Indeed, you're right :)